data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	"Mixed up the cargo, they have. His own longcow ranch on <destination>, a cousin of mine has," she says. "Sent his batch of vaccines along with my own, they have."`
+			`	They contact the cousin in question, and he asks that you transport his own part of the vaccines to <destination>, where he will pay you <payment>.`
+				accept
+	on complete
+		"coalition jobs" ++
+		payment 193000
+		conversation
+			`You land and meet up with the cousin of the Corral of Meblumem rancher, and he pays you <payment>.`
+			`	"Mix up our batches due to our family name, the doctors always do!" He grumbles as the vaccines are being removed from your cargo hold.`
+
+
+
+mission "Arachi Orphans"
+	minor
+	name "Arachi Orphans"
+	description "Bring these 4 Arachi orphans, their caretaker, and their <cargo> to <destination>, where they will join a foster home."
+	passengers 5
+	cargo "Orphanage Belongings" 2
+	to offer
+		has "Coalition: Contributor: done"
+	source
+		attributes arach
+	destination "Chosen Nexus"
+	on offer
+		conversation
+			`An elderly Arach beckons you as you enter the spaceport. He is accompanied by four Arachi children, as they all stand close to a large pile of luggage.`
+			`	He explains that he used to run an orphanage here, but now has gotten too old for the job, and has arranged for the four kids to be transferred to a foster home on <destination>.`
+			`	"Take care of them, the Saryds there will," he says, patting the head of the youngest and shortest of the kids. "Retiring, I am, but abandon them, I will not. An apartment close to the foster home, I will purchase."`
+			`	He offers to pay you <payment> to bring the five of them, and their <cargo> to <destination>.`
+			choice
+				`	"Of course I'll help. Here, let me help you get the luggage into my ship."`
+					accept
+				`	"I hope things work out for you, but I'm actually not going to stay in or fly around Coalition space for long now."`
+					decline
+	on complete
+		"coalition jobs" ++
+		payment 268000
+		conversation
+			`You help the old Arach caretaker and some workers take the luggage out of your ship, then head to the orphanage.`
+			`	A Saryd couple runs the place, and greet you and the old Arach when you get there. Aside from a few Saryd kids, you notice the place already has an Arach child, and some Kimek too.`
+			`	After you make sure all their belongings are inside the foster house, the Arachi children head on inside, to get acquainted with their new family. Their caretaker thanks you and hands you <payment> before heading inside himself.`
+
+
+
+mission "Arachi Cartoonist 1"
+	minor
+	name "A New Design"
+	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs for his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
```

Comment:
```suggestion
	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs in his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
```
Changed to avoid `for` used twice close together.